controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobProfile.java

```diff
@@ -68,7 +68,10 @@
 
     development(EnumSet.of(deployReal,
                            installReal,
-                           copyVespaLogs));
+                           copyVespaLogs)),
+
+    developmentDryRun(EnumSet.of(deployReal,
+                                 copyVespaLogs));
```

Comment:
Won't be any logss to copy either, since there will never be a logserver.

Reply:
Ah, good point, will fix